index.html

```diff
@@ -72,11 +71,11 @@ <h1 id="projectsText">Projects</h1>
 					<div id="projectsNotice">
 						<p>
 							Not much here yet, I’ve mostly been focusing on smaller projects<br>
-							and learning stuff. I’m still trying to figure out what area of CS i<br>
+							and learning stuff. I’m still trying to figure out what area of CS I<br>
 							enjoy the most, having fiddled with scripting for various games,<br>
 							fullstack web development,<br>
 							Web Application Security,<br>
-							a *bit* (if you get me) of x86 Assembly,<br>
```

Comment:
This was a pun, probably should be kept to try and keep a funny attitude

Reply:
Maybe replace it with something like
"a bit (if you pardon the pun) of x86 Assembly,"
it reads better